### PR TITLE
onlyRunIds option

### LINKED ISSUE #67: Request: Extend configuration options.
Hi! Thanks for the package. I have found it very useful to create a test run based on the cases that actually run not all of them. Could you extend the configuration options so the `data` looks like that?

```
if (config.plan) {
	if (config.plan.existingPlanId) {
		const data = {
			suite_id: suiteId,
			name: runName,
			include_all: false,
			config_ids,
			case_ids: ids,
			runs: [{
				include_all: false,
				case_ids: ids,
				config_ids
			}]
		};
		const res = await testrail.addPlanEntry(config.plan.existingPlanId, data);
		runId = config.runId ? config.runId : res.runs[0].id;
```

### PR DESCRIPTION
**Pull request by wjaros**
resolve #67 

This PR adds option to not include all ids while creating test run within an existing plan, instead of that you can create a run with the test cases that actually run

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -181,8 +181,9 @@ module.exports = (config) => {
 					const data = {
 						suite_id: suiteId,
 						name: runName,
-						include_all: true,
+						include_all: !config.plan.onlyRunIds,
```
Comment (PeterNgTr) [inline index.js:184]: shall `onlyCaseIds` sound more informative?

File: index.js [hunk 2]
```diff
@@ -181,8 +181,9 @@ module.exports = (config) => {
...
 						config_ids,
+						case_ids: ids,
```
Comment (PeterNgTr) [inline index.js:186]: do we always need `case_ids` here if `include_all` is `true`?

Comment (wjaros) [reply index.js:186]: we do not, but as far as I am concerned it does not change anything.

Comment (PeterNgTr) [reply index.js:186]: could we update the code like adding `case_ids` only if `include_all` is false?

Comment (wjaros) [reply index.js:186]: yup, but as long as `include_all` is `true` the `case_ids` is not being considered:
image

Comment (PeterNgTr) [reply index.js:186]: sounds good!

**Comment by PeterNgTr** [comment]
Thanks @wjaros !

**Comment by PeterNgTr** [comment]
`codeceptjs-testrail@1.8.13`released @wjaros

**Comment by wjaros** [comment]
Thanks! I was just about provide requested solution with condition ;)

**Comment by PeterNgTr** [comment]
shall be fine, we can always have room for improvements, tho :D

**Pull request merged**